code/modules/research/xenoarchaeology/tools/suspension_generator.dm

```diff
@@ -127,7 +127,7 @@
 				if(anchored)
 					activate()
 				else
-					to_chat(usr, "<span class='warning'>You are unable to activate [src] until it is properly secured on the ground.</span>")
+					to_chat(usr, "<span class='warning'>You are unable to activate \the [src] until it is properly secured on the ground.</span>")
```

Comment:
replace lines 125-132 with toggle(usr)

Reply:
Just to be sure, toggle(usr), not toggle(user)?